code_challenges/adhoc/insert.py

```diff
@@ -0,0 +1,15 @@
+def insert_target(target, num_array, low=0, high=None):
```

Comment:
Nice, are the low and high kwargs here because you attempted this recursively first?

Reply:
Yep! Haha!! I absolutely did! When I finally got it working, I definitely thought about removing them, BUT I was afraid I might cause some headaches in the future if I ever wanted to reuse the code for something else.